packages/redux-subspace/src/actions/reduxSpecific.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright 2017, IOOF Holdings Limited.
```

Comment:
```suggestion
 * Copyright 2018, IOOF Holdings Limited.
```

Reply:
Github wont let me commit this, but yes.